examples/01_getting_started.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+from haystack.document_stores import InMemoryDocumentStore
+from haystack.nodes import AnswerParser, PromptNode, PromptTemplate, BM25Retriever
+from haystack.pipelines import Pipeline
+from haystack.pipelines.standard_pipelines import TextIndexingPipeline
+from haystack.utils import fetch_archive_from_http
+from haystack.utils import print_answers
+
+
+def getting_started():
+    # This getting_started shows you how to use LLMs with your data, a technique called Retrieval Augmented Generation - RAG
+
+    # We are model agnostic :) Here, you can choose from: "Anthropic", "Cohere", "Huggingface", "OpenAI".
+    provider = "OpenAI"
+    # Please add an API key matching the provider.
+    API_KEY = "ADD KEY HERE"
+
+    # Loads good default pipelines for each provider.
+    indexing_pipeline, query_pipeline = load_pipelines(provider=provider, API_KEY=API_KEY)
+
+    # Downloads and adds a few Game of Thrones wiki articles to haystacks database.
+    # You can also provide a folder with your local documents. Be aware that some of your data will be sent to external APIs!
+    add_example_data(indexing_pipeline, dir="data/getting_started")
+
+    result = query_pipeline.run(query="Who is the father of Arya Stark?")
+
+    # This prints only the answer.
+    # For details, like which documents were used to generate the answer, look into the <result> object
+    print_answers(result, details="medium")
+
+
+def load_pipelines(provider, API_KEY):
```

Comment:
Nitpick: `load_pipelines` makes it sound like it's loading them from somewhere. This function rather "builds" the pipelines. Maybe we can change the name?

I woudn't like people to get confused with `load_from_yaml`

Reply:
(also, `load_pipelines` happens to be the name of Haystack's v2 function that replaces `load_from_yaml`, so that's extra confusing)